x/rollup/keeper/deposits.go

```diff
@@ -108,7 +108,7 @@ func (k *Keeper) MintETH(ctx sdk.Context, addr sdk.AccAddress, amount sdkmath.In
 		return fmt.Errorf("failed to send deposit coins from rollup module to user account %v: %v", addr, err)
 	}
 
-	// TODO: move event emission to msg_server.go
+	// TODO: only emit sdk.EventTypeMessage once per message
```

Comment:
Noticed a bug with the message emission for `ApplyL1Txs`, I updated https://github.com/polymerdao/monomer/issues/162 to reflect this